tests/integration/update_cluster/aws-lb-controller/data/aws_launch_template_master-us-test-1a.masters.minimal.example.com_user_data

```diff
@@ -121,6 +121,8 @@ ensure-install-dir
 
 cat > conf/cluster_spec.yaml << '__EOF_CLUSTER_SPEC'
 cloudConfig:
+  awsEBSCSIDriver:
+    version: v1.0.0
```

Comment:
are we enabling the driver by default? I'm surprised to see the version set here if its not enabled.

Reply:
Spotted this one too. No, we are not. It should look better now.